list.txt

```diff
@@ -71,6 +71,7 @@ freenitros.ru
 gave-nitro.com
 get-nitro.net
 giftdiscord.info
+giveaweys.com
```

Comment:
Agreed, though for sake of clarity I think it would be best to add it with the `discord` subdomain - so it'd be added as `discord.giveaweys.com` - if you don't mind updating your commit that'd be great, otherwise I'll edit it in with contribution credits in the next day or so.